yaml-schema.json

```diff
@@ -0,0 +1,138 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "title": "Meta yaml",
+    "description": "Validate the meta yaml file for an nf-core module",
+    "type": "object",
+    "properties": {
+        "name": {
+            "type": "string",
+            "description": "Name of the module"
+        },
+        "description": {
+            "type": "string",
+            "description": "Description of the module"
+        },
+        "keywords": {
+            "type": "array",
```

Comment:
Can you specify a minimum number of elements in the array? Might be good to stop people just putting `samtools` or something not very useful (I would suggest at least 3)